component/distributed_cache/distributed_cache.go

```diff
@@ -442,12 +463,19 @@ func (dc *DistributedCache) CreateFile(options internal.CreateFileOptions) (*han
 
 	return handle, nil
 }
+
 func (dc *DistributedCache) OpenFile(options internal.OpenFileOptions) (*handlemap.Handle, error) {
 	var dcFile *fm.DcacheFile
 	var handle *handlemap.Handle
 	var err error
 
-	isAzurePath, isDcachePath, rawPath := getFS(options.Name)
+	isAzurePath, isDcachePath, isDebugPath, rawPath := getFS(options.Name)
+
+	if isDebugPath {
+		options.Name = rawPath
+		return debug.OpenFile(options)
+	}
```

Comment:
till we have some practical usecase for allowing writes, let's fail open() with access denied